scenes/game_elements/characters/enemies/throwing_enemy/components/throwing_enemy.gd

```diff
@@ -8,7 +8,7 @@ extends CharacterBody2D
 enum State { IDLE, WALKING, ATTACKING, DEFEATED }
 
 const PROJECTILE_SCENE: PackedScene = preload(
-	"res://scenes/quests/lore_quests/quest_001/2_ink_combat/components/projectile/projectile.tscn"
+	"res://scenes/game_elements/props/projectile/projectile.tscn"
```

Comment:
It doesn't have to be changed here but I noticed that in the Sokoban code, @PhoenixStroh used `uid://` paths for paths embedded in GDScript source code, presumably so that they don't have to be updated when the source tree is rearranged. It's a bit more opaque, but you can hover to see what the `uid://` resolves to and ctrl-click to jump to it. I did the same in the splash screen when I inserted the title screen after it. We might try to get into the habit of doing this in future.

![image](https://github.com/user-attachments/assets/2ec19094-6224-4822-ab84-24b49c443dfe)

Reply:
Oh good point, I didn't know about it. Yes, I'll switch to uid paths.